OpenRA.Mods.Common/TraitsInterfaces.cs

```diff
@@ -28,6 +28,7 @@ public interface IPlaceBuildingDecoration
 	}
 
 	public interface INotifyAttack { void Attacking(Actor self, Target target, Armament a, Barrel barrel); }
+	public interface INotifyCharging { void Charging(Actor self); }
```

Comment:
This interface would be more flexible if it also provided the target that the actor is charging to attack.